src/test/resources/features/tasks/create_task.feature

```diff
@@ -0,0 +1,24 @@
+Feature: Create task
+
+  @CreateList @DeleteTask
+  Scenario: As a user I want to create a list for tracking an objective
+    Given I set the task with request endpoint to /list/{list_id}/task
+    When I set the request body with following values:
+      | name            | My Task created                                             |
+      | description     | New Task Description                                        |
+      | assignees[0]    | 183                                                         |
+      | tags[0]         | tag name 1                                                  |
+      | status          | complete                                                    |
+      | priority        | 3                                                           |
+      | due_date        | 1508369194377                                               |
+      | due_date_time   | false                                                       |
+      | time_estimate   | 8640000                                                     |
+      | start_date      | 1567780450202                                               |
+      | start_date_time | false                                                       |
+      | notify_all      | true                                                        |
+      | parent          | null                                                        |
+      | links_to        | null                                                        |
+      | Custom fields   | src\test\resources\json\tasks\custom_fields_by_defects.json |
```

Comment:
Is there another way to manage this path, have a path hardcoded makes the scenario hard to read

Reply:
The main reason the user or whatever wants to see the contents of custom_fields_by_defects.json is to know where it is.